src/control/cmd/dmg/pretty/storage_nvme_test.go

```diff
@@ -31,10 +45,10 @@ func TestPretty_PrintNVMeController(t *testing.T) {
 				storage.MockNvmeController(2),
 			},
 			expPrintStr: `
-NVMe PCI     Model   FW Revision Socket ID Capacity 
---------     -----   ----------- --------- -------- 
-0000:01:00.0 model-1 fwRev-1     1         2.0 TB   
-0000:02:00.0 model-2 fwRev-2     0         2.0 TB   
+NVMe PCI     Model   FW Revision Socket Capacity Role(s) Rank 
+--------     -----   ----------- ------ -------- ------- ---- 
+0000:01:00.0 model-1 fwRev-1     1      2.0 TB   ???     None 
+0000:02:00.0 model-2 fwRev-2     0      2.0 TB   ???     None 
```

Comment:
I'm a little confused as to why the "???" string is coming out in the tests, as you mentioned above it shouldn't be displayed. Seems like we should be displaying "N/A". I don't fully understand why that isn't the default if no bits are set.

Reply:
The thinking was to differentiate between the situation where no SMD is referenced by the ctrlr and where there is a SMD with no bits set. In the end this isn't really necessary so it now reports "N/A" for both those cases. I wanted to avoid the situation where "N/A" was reported incorrectly in MD-on-SSD mode just because the ctrlr is missing a SMD reference but in the end I think this is covered sufficiently in unit testing.